src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -100,7 +101,7 @@ export const JobLogsContainer = props => {
   };
 
   const handleRowClick = (e, row) => {
-    if (row?.progress?.failed) {
+    if (row?.status !== JOB_EXECUTION_STATUSES.COMPLETED) {
```

Comment:
I think we can assume that status we will have for sure.
```suggestion
    if (row.status !== JOB_EXECUTION_STATUSES.COMPLETED) {
```